rbi/sorbet/builder.rbi

```diff
@@ -39,3 +39,8 @@ class T::Private::Methods::DeclBuilder
   sig {params(arg: T.untyped).returns(T::Private::Methods::DeclBuilder)}
   def checked(arg); end
 end
+
+class T::Private::Methods::DeclarationBlock
```

Comment:
Curious what's your mental model of what this name means.

Reply:
We might want to give it a more specific name to make people think carefully about lurking things into it.

Reply:
> We might want to give it a more specific name to make people think carefully about lurking things into it.

You mean like I did? 😅 

I suppose I'd say that `DeclarationBlock` is something like "the ingredients to produce a correctly checked method from a user-defined method".